src/main/webapp/assets/modules/common/feedback/feedbackDirective.js

```diff
@@ -27,23 +33,28 @@ techlooper.directive("feedbackForm", function (apiService, $timeout) {
       }
 
       scope.send = function(){
-        scope.composeEmail.content = $('.summernote').code();
-        if(scope.composeEmail.content == '<p><br></p>' || scope.composeEmail.content == ''){
+        if(scope.feedbackContent == undefined || scope.feedbackContent == ''){
```

Comment:
Use '===' to compare with 'undefined'.<br>Mixed double and single quotes.<br>Use '===' to compare with ''.